x/dsmr/node_test.go

```diff
@@ -768,196 +768,173 @@ func TestNode_NewBlock_IncludesChunkCerts(t *testing.T) {
 
 	tests := []struct {
 		name      string
-		chunks    []chunk
-		parent    Block
-		timestamp int64
+		chunks    func(parent Block) []chunk
+		timestamp func(parent Block) int64
 		wantErr   error
 	}{
 		{
 			name: "no chunk certs",
-			parent: Block{
-				ParentID:  ids.GenerateTestID(),
-				Height:    1,
-				Timestamp: 1,
-				blkID:     ids.GenerateTestID(),
+			timestamp: func(parent Block) int64 {
+				return parent.Timestamp + 1
 			},
-			timestamp: 2,
 			// TODO should we be able to build empty blocks?
 			wantErr: ErrNoAvailableChunkCerts,
 		},
 		{
 			name: "timestamp equal to parent",
-			parent: Block{
-				ParentID:  ids.GenerateTestID(),
-				Height:    1,
-				Timestamp: 1,
-				blkID:     ids.GenerateTestID(),
+			timestamp: func(parent Block) int64 {
+				return parent.Timestamp
 			},
```

Comment:
It doesn't make sense for us to specify the parent in the test vector because the parent state is now defined by the test node helper.